ui/components/ui/tabs/tab/tab.component.js

```diff
@@ -25,7 +26,7 @@ const Tab = (props) => {
         onClick(tabIndex);
       }}
     >
-      <button>{name}</button>
+      <button>{t(tabKey)}</button>
```

Comment:
Unfortunately, we try to avoid doing translations like this, where the string is passed by reference as a variable, as opposed to directly passing a string literal. The reason for this (which you would have had no way of knowing because this is not documented) is that we have tooling which detects when translations are unused, or when they are undefined in in our translation file, but that tooling only works if we explicitly pass string literals to the translation function

Reply:
Actually the data flow is rendered like this, hope this makes sense: 

- We are in "Activity" tab, current `defaultHomeActiveTabName` is `Activity`
- We switch language, and close the settings
- `defaultHomeActiveTabName` is still `Activity` while the `name` is `activité` for example

=> this results in in consistency and wrong `activeTabIndex`

2 ways i think could be a fix
1. use the key of language as tabKey and translation in the end, like the solution i provided, which doesn't work actually
2. reverse the language by key and use the key in tabs component to compare, which is not very possible.
Open to more thoughts !